austation/code/modules/hydroponics/grown.dm

```diff
@@ -1,3 +1,18 @@
+/obj/item/reagent_containers/food/snacks/grown
+	var/mutable_appearance/grown_overlay //Used for 'Floral Spines' plant gene
+	var/mob/living/target
+
+
+/obj/item/reagent_containers/food/snacks/grown/Initialize(mapload, obj/item/seeds/new_seed)
+	. = ..()
+
+	if(seed.get_gene(/datum/plant_gene/trait/spines))
+		embedding = EMBED_HARMLESS
+		embedding["embed_chance"] = 300 //300 is better than 100 ;)
+		updateEmbedding()
+	
+	grown_overlay = mutable_appearance(icon, icon_state)
```

Comment:
what's the reason you need a dedicated variable to store this? why not just generate it at the time of embed?